### PR TITLE
Practice app forgot password

### PR DESCRIPTION
**Pull request by MEHMETTEMIZEL**
Please Make a code review and merge if there is not any problem. If there are sone things that must change please write on the linked issue about Forgot Password.

Please control these criteria.(If you want you can add extra criteria.)
- [x] Documentation
- [x] Readability
- [x] Code works correctly or not?
- [x] Tests

### CODE REVIEW

File: practice-app/platon_api/rest_api/forgot_password/forgot_password.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from rest_api.forgot_password.forms import ForgotForm
+import re,copy, hashlib, random, json, string, requests
+from django.shortcuts import render, get_object_or_404
+from django.http import HttpResponse
+from rest_api.models import RegisteredUser
+from rest_framework.response import Response
+
```
Comment (meltemarsl) [inline practice-app/platon_api/rest_api/forgot_password/forgot_password.py:7]:
```suggestion

#I think it is not necessary to import them: re,copy, random, json, requests, render, Response

```
```

File: practice-app/platon_api/platon_api/settings.py [hunk 2]
```diff
@@ -1,13 +1,13 @@
 """
 Django settings for platon_api project.
 
-Generated by 'django-admin startproject' using Django 3.0.5.
+Generated by 'django-admin startproject' using Django 2.2.12.
```
Comment (Hryurt) [inline practice-app/platon_api/platon_api/settings.py:4]: You can check all requirements by using requirements.txt in merged folder.

File: practice-app/platon_api/platon_api/urls.py [hunk 3]
```diff
@@ -14,8 +14,12 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
```
Comment (Hryurt) [inline practice-app/platon_api/platon_api/urls.py:17]: you do not need to use include

**Comment by alperendivriklioglu** [comment]
After the review of my teammates, I could not find a missing part. Everything looks OK.

**Pull request merged**